contact.md

```diff
@@ -131,19 +140,19 @@ Please send separate emails for each problem. If you send an email about a
 new problem by replying to an old issue, it may delay our response.
 If you reply to an old issue that has been closes, we will not see the reply.
 
-## Can I call you?
+### Can I call you?
 
-Nope. We do not have a phone number, and we do not have the capacity to
```

Comment:
I liked "nope"! 😅

Reply:
I know, but it sounds like we are insensitive. :-D